packages/server/graphql/mutations/helpers/notifications/getSummaryText.ts

```diff
@@ -39,7 +40,11 @@ const getSummaryText = (meeting: Meeting) => {
       'comment'
     )}.`
   } else if (isMeetingTeamPrompt(meeting)) {
-    return 'TODO: Implement teamPrompt summary text'
+    const responseCount = (await getTeamPromptResponsesByMeetingId(meeting.id)).filter(
```

Comment:
+1 this will do db hit for every integration configured, we may want to use similar pattern as retro ie. store the response count in the meeting object that'd be set when meeting ends and just read it here

Reply:
We decided against that pattern as part of CR in https://github.com/ParabolInc/parabol/pull/6529. We could reduce the DB calls by moving `getTeamPromptResponsesByMeetingId` to a dataloader, but I don't think doing so is important enough for this PR, especially since there's already an index for this table on `meetingId`.